installing/installing_gcp/installing-gcp-customizations.adoc

```diff
@@ -37,6 +37,10 @@ include::modules/installation-configuration-parameters.adoc[leveloffset=+2]
 
 include::modules/installation-gcp-config-yaml.adoc[leveloffset=+2]
 
+== Additional resources
```

Comment:
I believe we're including an ID for these in assemblies, just like the H1s.

Reply:
For now, I'm going to keep this as-is for consistency, since the majority of installation generic H2s (i.e., Next steps, Prereqs, Additional resources) do not have them at the moment.